### PR TITLE
Preprocess fixes

### PR DESCRIPTION
**Pull request by fromanirh**

### CODE REVIEW

**Comment by fromanirh** [comment]
@ksimon1 please review especially https://github.com/MarSik/kubevirt-ssp-operator/pull/14/commits/a2ee009795c42ef8f951f889fafc56b67eb3996a - not sure we want to fail or not with empty patch.yaml

File: build/preprocess_template.py [hunk 1]
```diff
@@ -91,6 +96,9 @@ def process_common_templates(commonTemplatesPath, rules):
   rules = None
   try:
     rules = load_rules(patchPath)
+  except MissingPatch:
+    # nothing to do, let's just bail out
+    sys.exit(0)
```
Comment (MarSik) [inline build/preprocess_template.py]: It might be good idea to log this.

Comment (fromanirh) [reply build/preprocess_template.py]: Good idea. On it.

**Comment by fromanirh** [comment]
works for me - actually, I can't build the operator without this patch.

**Pull request merged**